packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -189,20 +191,29 @@ interface UserMenuProps {
 ```
  * @param props.account Object that contains information about the active user. {firstName: 'string', lastName: 'string', email: 'string'}
  * @param props.activeProduct MongoDB product that is currently active: ['cloud', 'university', 'support'].
- * @param props.onLogout Callback invoked after the user clicks log out.
+ * @param props.onLogout Callback fired when a user logs out
  * @param props.onProductChange Callback invoked after the user clicks a product.
  * @param props.hosts Object where keys are MDB products and values are the desired hostURL override for that product, to enable `<UserMenu />` to work across all environments.
  * @param props.urls Object to enable custom overrides for every `href` used in `<UserMenu />`.
  */
 function UserMenu({
   account,
   activeProduct,
-  onLogout = () => {},
+  onLogout: onLogoutProp,
   onProductChange = () => {},
   urls: urlsProp,
   hosts: hostsProp,
 }: UserMenuProps) {
   const hosts = defaultsDeep(hostsProp, hostDefaults);
+  const onElementClick = useOnElementClick();
```

Comment:
Does this work?

Also ideally the prop gets called with the same API as everything else. 

```suggestion
  const onElementClick = useOnElementClick() ?? onLogoutProp;
```

Reply:
the prop doesn't need the type of onElementClick though

Reply:
Oh, didn't realize it was still named onLogout. Rightio, SGTM.

Reply:
👍